### PR TITLE
pyproject: Fix broken 32-bit support

### PR DESCRIPTION
**Pull request by johan-bjareholt**
Found in this forum post

https://forum.activitywatch.net/t/32-bits-package-or-windows-compiling-how-to/643/6

### CODE REVIEW

**Comment by johan-bjareholt** [comment]
@ErikBjare Why isn't travis running?
Would like the CI to try to build it on windows

File: pyproject.toml [hunk 1]
```diff
@@ -10,12 +10,19 @@ aw-watcher-afk = "aw_watcher_afk:main"
...
-pypiwin32 = {version = "^223", platform = "win32"}
-pyhook = {url = "https://github.com/ActivityWatch/wheels/raw/master/pyHook-1.5.1-cp36-cp36m-win_amd64.whl", platform = "win32" }
+python-xlib = { git = "https://github.com/python-xlib/python-xlib.git", platform = "linux" }
+pyuserinput = { git = "https://github.com/PyUserInput/PyUserInput.git", platform = "linux", rev = "ac2d4c7a7f4b1a72e70b1a2ef8925d5312fb12bc" }
+pypiwin32 = { version = "^223", platform = "win32" }
+pyhook = {
+    url = "https://github.com/ActivityWatch/wheels/raw/master/pyHook-1.5.1-cp36-cp36m-win_amd64.whl",
+    platform = "win32", platform_machine = "x86_64"
+}
+pyhook = {
```
Comment (ErikBjare) [inline pyproject.toml:22]: Is it even valid to have two pyHook entries like this? Wouldn't they collide in the resulting TOML object?

**Comment by ErikBjare** [comment]
I also have no idea why Travis isn't running. I could look into it after the weekend.

**Comment by johan-bjareholt** [comment]
> Not sure this'll work as you intended.

I'm not sure either, but I need to either build it on windows or have a functioning CI to figure that out.

**Comment by ErikBjare** [comment]
I was certain that I had added Github actions to this repo, and I have: https://github.com/ActivityWatch/aw-watcher-afk/pull/39

However, that PR is stalled due to the therein mentioned issue with poetry.

**Comment by johan-bjareholt** [comment]
Not able to solve without poetry fixing the following issue, closing

https://github.com/python-poetry/poetry/issues/1616

**Pull request closed without merging**